visualization/app/codeCharta/ui/customConfigs/customConfigs.component.html

```diff
@@ -67,6 +67,7 @@
 							ng-click="$ctrl.removeCustomConfig(customConfig.id)"
```

Comment:
I think it is possible to prevent collapsing the list on remove.
You could try to append a call to `$event.stopPropagation();` within the `ng-click` attribute (untested).